### ISSUE TITLE
Call for input: where do we go from here? 

### ISSUE THREAD

**Issue by jbrowning** [pineconellc/angular-foundation#311]
**TL/DR**
* We'd like your help in deciding the future of Angular Foundation
* If the community decides that Angular Foundation should continue, we're looking for new ownership to take over the project

Hello everyone,

As you've probably noticed, this project has been dormant for quite a while. We initially created Angular Foundation to help build https://pinecone.com and the short story is that the project in its current state has been "good enough" for our needs. We are a tiny bootstrapped team and have since moved our focus on to new [products and projects](https://yalabot.com). With such limited resources, we simply don't have the time to give Angular Foundation the love that it deserves. **SO**, where the project goes from here depends on **YOU**.

Since we created Angular Foundation in 2013, several things have happened:

* ZURB created [Foundation for Apps](http://foundation.zurb.com/apps.html), their own Angular-based UI framework
* Foundation (now called [Foundation for Sites](http://foundation.zurb.com/sites.html)) has reached Version 6
* Angular 2.0 has been released
* ES2015/ES6/Whatever has reached enough of a critical mass to become a "standard" (such as it is in JS-land)

These events have all combined to make the future of Angular Foundation less clear. Should it be:

1. Deprecated in favor of Foundation for Apps or some other framework?
2. Placed in maintenance mode with its current Angular/Foundation support? (Foundation 5, Aangular 1.x)
3. Updated to work with Foundation 6 but pegged to Angular 1.x for now?
4. Updated to work with Foundation 6 AND updated for Angular 2.0?
5. Updated to work with Foundation 6 AND updated to TypeScript AND updated for Angular 2.0?
6. Something else?

If the answer is anything other than :one:, we'd like to transfer ownership of the project so that it can continue onwards. Please let us know your thoughts.

We truly appreciate the love and support that this project has received from the community. Wherever we go from here, it's been a great ride.

**Comment by terotil** [comment]
I'm working with a codebase which uses Angular Foundation. In case Angular Foundation will continue on Angular 1.x, (options 2 and 3) our team is interested in devoting time to participate in maintenance and development.

**Comment by kball** [comment]
HI @jbrowning,

I'm an engineering lead at ZURB and as of relatively recently leading the Foundation project. 

A couple things of note to help with this decision:

1. After experimenting with Foundation for Apps ZURB is moving in a direction of working more with community implementations for things like Angular, React, etc.  We don't think Foundation should be 'picking a winner' with regards to JavaScript frameworks, but rather working with each of these communities. This is in the process of getting formalized and will be communicated broadly soon, but is important for this decision.

2.  There is already fork of this repo that has been updated for Angular 1.5 and Foundation 6 here: https://github.com/circlingthesun/angular-foundation-6

3. As far as I know, there are no good Angular2 ports as of yet, but we'd be very interested in seeing that happen and would be happy to help support.

Regards

**Comment by lukens** [comment]
Hi, Foundation for apps seems to have been pretty much abandoned by Zurb, leading to [a community driven fork](https://github.com/base-apps/angular-base-apps). I'm not sure what the future roadmap is for this though, and wether it will go anywhere moving forwards, or will essentially just be purely for maintenance of the current version.

I do wish Zurb would make a more public announcement around the status and future of Foundation for apps, though from the "Future of Foundation" section on their [Birthday Page](http://foundation.zurb.com/foundationturns5.html), there is no mention of Foundation for Apps.

As @kball has mentioned, there is also a Foundation 6 fork of this repository.

I found out all of the above when searching for whether there was going to be an Angular 2 version of Foundation for apps. It seems unlikely that this is ever going to happen, and I think what I'd like to see most at this stage is an Angular 2 library for Foundation 6.

As for the future of this project, it may be worth tagging in people from the [base-apps](https://github.com/base-apps) and the [Foundation 6 fork](https://github.com/circlingthesun/angular-foundation-6) to get input from them.

**Comment by terotil** [comment]
Our team is switching over to https://circlingthesun.github.io/angular-foundation-6/
